src/styled/customStyles.scss

```diff
@@ -117,4 +117,14 @@ $HEADER-HEIGHT: 4.125rem;
 #bee-modal-root {
   position: relative;
   z-index: 10000;
+}
+
+.bee-directions-showing {
+  .gm-style-iw {
+    opacity: 0.75;
```

Comment:
we could add `transition: all 0.2s ease-in-out;` to smoothen transition

Reply:
Nice touch! Added